frontend/src/ui-components/AcmTable/AcmManageColumn.tsx

```diff
@@ -50,7 +60,7 @@ export function AcmManageColumn<T>({
   }
 
   return (
-    <ToolbarItem>
+    <ToolbarItem spacer={toolbarItemSpacerSetting}>
```

Comment:
I don't think the `<ToolbarItem>` component should be included in this component. I would move it up into the table. This component shouldn't be restricted to only appearing in a toolbar.